modular_skyrat/master_files/code/modules/power/lighting/light_mapping_helpers.dm

```diff
@@ -0,0 +1,8 @@
+// Kneecapping light values every light at a time.
+/obj/machinery/light/dim
+	bulb_colour = "#FFDDBB"
+	bulb_power = 0.4
+
+/obj/machinery/light/small
+	bulb_colour = "#FFDDBB"
+	bulb_power = 0.45
```

Comment:
I would like to see be made darker. As it stands, night shifts are, almost visually similar to regular lighting.
Alternatively, fiddle with the color AND bulb power

Reply:
I'll just change the night shift values. No point in faffing around with the light settings that'll only just make the issue more obvious.